dspace-api/src/main/java/org/dspace/core/HibernateDBConnection.java

```diff
@@ -184,17 +184,19 @@ private void configureDatabaseMode() throws SQLException {
                 }
             }
 
+            // ITEM
             if (entity instanceof Item) {
                 Item item = (Item) entity;
 
-                if(Hibernate.isInitialized(item.getSubmitter())) {
-                    uncacheEntity(item.getSubmitter());
-                }
+                //DO NOT uncache the submitter. This could be the current eperson. Uncaching could lead to
```

Comment:
If the problem is that the commiter might be the current eperson, we should check that and uncache the submitter otherwise. I'm a bit afraid that someone loads a lot of items for some batch processing, uncaches those items but does not know that there are some epersons loaded, he did not uncache properly. The next question would then be: If I would want to uncache the submitter and the item in which order should I do it?